packages/react-components/react-provider/src/components/FluentProvider/FluentProvider.types.ts

```diff
@@ -19,18 +20,23 @@ export type FluentProviderProps = Omit<ComponentProps<FluentProviderSlots>, 'dir
   targetDocument?: Document;
 
   theme?: PartialTheme;
+
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  overrides_unstable?: OverridesContextValue; // FIXME: better type
 };
 
 export type FluentProviderState = ComponentState<FluentProviderSlots> &
   Pick<FluentProviderProps, 'targetDocument'> &
   Required<Pick<FluentProviderProps, 'dir'>> & {
     theme: ThemeContextValue;
     themeClassName: string;
+    overrides: OverridesContextValue;
```

Comment:
This is a breaking change for anyone who uses our unstable hooks (as `overrides` in mandatory now), right?

Reply:
Only if they explicitly type an object which they create, it should be ok otherwise, right?
What would you suggest?

Reply:
Hooks are unstable, so let's leave it as it is.